ui/app/styles/utilities/_index.scss

```diff
@@ -1,2 +1,3 @@
 @import 'animations';
 @import 'focus-ring';
+@import 'sr-only';
```

Comment:
[dust] tbh that `sr-only` class/file-name is not very descriptive. I'm still wondering what it refers to 🤔

Reply:
Good call. I’ve switched over to `pds--visuallyHidden` (which I couldn’t remember how to invoke in the moment).